honeybee_vtk/load_config.py

```diff
@@ -0,0 +1,236 @@
+        legend.label_count = legend_params.label_count
+    else:
+        legend.label_count = None
+
+    legend.decimal_count = legend_params.decimal_count
+    legend.preceding_labels = legend_params.preceding_labels
+
+    label_params = legend_params.label_parameters
+    legend.label_parameters = Text(
+        label_params.color, label_params.size, label_params.bold)
+
+    title_params = legend_params.title_parameters
+    legend.title_parameters = Text(
+        title_params.color, title_params.size, title_params.bold)
+
+
+def _get_model_with_point_in_time_data(
+        validation: bool, data: DataConfig, model: Model, grid_type: str, scene: Scene,
+        legend: bool) -> Model:
+    """Get a model with point_in_time data loaded.
+
+    Args:
+        validation: If True, the data will be validated.
+        data: A DataConfig object from the config file.
+        model: A honeybee-vtk model.
+        grid_type: A string indicating whether the model has points and meshes.
+        scene: A honeybee-vtk scene.
+        legend: A boolean to indicate whether the legend should be loaded.
+
+    Returns:
+        A honeybee-vtk model with point_in_time data loaded.
+    """
+    folder_path = pathlib.Path(data.path)
+    identifier = data.identifier
+    object_type = data.object_type
+
+    if validation:
+        _validate_simulation_data(data, model, grid_type)
+    model = _load_point_in_time_data(
+        folder_path, identifier, object_type, model, grid_type)
+    # Legend will only be used in export-images command
+    if legend and data.legend_parameters:
+        _load_legend_parameters(data, model, scene)
+    return model
+
+
+def load_config(json_path: str, model: Model, scene: Scene,
+                validation: bool = False, legend: bool = False,
+                time_series: bool = False, hbjson: str = None) -> Model:
+    """Mount data on model from config json.
+
+    This functions outputs a model or a time series folder based on whether the time
+    series flag is used.
+
+    Args:
+        json_path: File path to the config json file.
+        model: A honeybee-vtk model object.
+        scene: A honeybee-vtk scene object.
+        validation: A boolean indicating whether to validate the data before loading.
+            Defaults to False.
+        legend: A boolean indicating whether to load legend parameters.
+            Defaults to False.
+        time_series: A boolean indicating whether to load time series data.
+            Defaults to False.
+        hbjson: File path to HBJSON. This is only used in the case of time series data.
+            Defaults to None.
+
+    Returns:
+        Either a honeybee-vtk model with data loaded on it or a time series folder.
+    """
+    try:
+        with open(json_path) as fh:
+            config = json.load(fh)
+    except json.decoder.JSONDecodeError:
+        raise TypeError(
+            'Not a valid json file.'
+        )
+    else:
```

Comment:
This comment is mostly for making the code pretty.

I would not use `else` here since there is no follow-up section in the code. if it doesn't create an exception then the rest of the code should be executed anyways.